tests/parser.js

```diff
@@ -60,9 +60,8 @@ require([
                         var mug = call("getMugByPath", "/data/state");
                         assert.equal(mug.__className, "Select");
                         var xml = call("createXML"),
-                            doc = $($.parseXML(xml)),
-                            instance = '<instance id="states" src="jr://fixture/item-list:state"></instance>';
-                        assert.operator(xml.indexOf(instance), ">", -1);
+                            doc = $($.parseXML(xml));
+                        assert.equal(doc.find("instance[id=states]").length, 1, xml);
```

Comment:
Just having the XML with no explanation as the error message seems bad. 

Looks like the convention is `instance not found: xml` elsewhere.

Personally I'm not a huge fan of having the entire xml in the error message if it's not a diff from expected/actual, but I suppose that's personal preference.